Account => Many2one Replace Account => Many2many

kuro088 opened the pull request:
Conciliación de multiples cuentas

Commit:
Account => Many2one Replace Account => Many2many

agb80 commented on account_easy_reconcile/easy_reconcile.py:
```diff
@@ -185,8 +184,7 @@ def _last_history(self, cr, uid, ids, name, args, context=None):
         'name': fields.char('Name', required=True),
-        'account': fields.many2one(
-            'account.account', 'Account', required=True),
+        'account': fields.many2many('account.account','reconcile_account_tbl','reconcile_id','account_id','Accounts to Reconcile',domain=[('reconcile','=',1)]),
```
Por favor respeta los 80 caracteres máximos a la hora de agregar lineas de código en python

Commit:
FIX: Method open_unreconcile

Commit:
MOD: Identation easy_reconcile.xml

agb80 commented:
Hay un problema con el registro de errores cuando los mensajes están en español. Pasos para reproducir el problema:

1.- Crea una BD nueva y carga en automático el idioma español es_MX
2.- Crea una factura y un pago por separado en el periodo actual
3.- Cierra el periodo en Procesamiento periodico >> Fin de periodo >> Cerrar un periodo
4.- Crea un nuevo asistente para conciliación automática asegurandote que las columnas Fecha de conciliación tengan seleccionada la opción Most recent move line
5.- Corre el proceso de conciliación automática

agb80 commented:
El error está en estas líneas: https://github.com/OpenPymeMx/bank-statement-reconcile/pull/1/files#diff-c1fe897527c8c14402ed407ae9dae6ceR295

agb80 commented:
Otro comentario, por la forma en como se están guardando los registros de las operaciones crea muchos históricos, uno por cada cuenta con lo que se pierde la utilidad de las funcionalidades de Mostrar elementos conciliados en la última ejecución.

Te dejo unas capturas de pantalla para que veas a lo que me refiero.

![captura de pantalla de 2015-08-13 13-06-32](https://cloud.githubusercontent.com/assets/911891/9257643/45a8ac44-41bc-11e5-9782-1d81a0ff252a.png)

![captura de pantalla de 2015-08-13 13-06-16](https://cloud.githubusercontent.com/assets/911891/9257636/3d44d65e-41bc-11e5-8192-8fb1ef148b4f.png)

NOTA: solo he ejecutado el asitente una sola vez, fíjate como el número de cuentas es igual al número de lineas en el historial

Commit:
ADD: ORM Exception

Commit:
FIX: encode utf8

Commit:
FIX: replace method str(error) per unicode(error) in file easy_reconcile.py

Commit:
ADD: Translate es_MX

Commit:
FIX:Method get_partial_rec

ManulReyes committed:
fix(account_easy_reconcile/base_reconciliation.py): Create the relation to load the easy_reconcile m

add the relation in base_reconciliation.py file allowed load properly easy_reconcile module
